### DISCUSSION TITLE
Clips dir with frigate 0.9x and above

### DISCUSSION THREAD

**Discussion by grembling22** [blakeblackshear/frigate#2481] [General]
Im hoping I just missed this. Is there a way to have the detection based clips stored in the clips directory? I use h265 to cut down on the storage space used, and on version 0.8 and below would just browse to that network share to review clips since edge and chrome cannot play them in the browser.

**Comment by blakeblackshear** [comment]
See release notes: https://github.com/blakeblackshear/frigate/releases/tag/v0.9.1

TLDR; there is no way to get clips in the clips folder

**Reply by grembling22** [reply]
Thanks. I did see the notes, and read through the docs. I was hoping someone had a workaround. Switching to h264 filled up almost 1TB on my network share with 2.5 days of recordings.
